repl/post.go

```diff
@@ -0,0 +1,116 @@
+package repl
+
+import (
+    "strconv"
+    "strings"
+    ishell "github.com/abiosoft/ishell/v2"
+    log "github.com/sirupsen/logrus"
+    "github.com/GridPlus/phonon-client/util"
+    "encoding/hex"
+)
+
+func postPhonons(c *ishell.Context) {
+  log.Debug("initiating postPhonons")
+  if ready := checkActiveCard(c); !ready {
+		return
+	}
+
+  c.Println("What is receiving card's public key hex?")
+  receivingPubKeyInput := c.ReadLine()
+	receivingCardPubKeybytes, err := hex.DecodeString(receivingPubKeyInput)
+  receivingCardPubKey, err := util.ParseECCPubKey(receivingCardPubKeybytes)
+  receivingCardPubKeyXbytes := receivingCardPubKey.X.Bytes()
```

Comment:
The function you're looking for here comes from geth. You can import as 
```ethcrypto "github.com/ethereum/go-ethereum/crypto"```
and then the call is ethcrypto.FromECDSAPub(*ecdsa.PublicKey)

However I think it would probably be best to pass around the *ecdsa.PublicKey object until you absolutely need to convert it to bytes, as it's the more specific and obvious type.